WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewModel.swift

```diff
@@ -699,6 +715,20 @@ private extension ProductFormViewModel {
     }
 }
 
+// MARK: Blaze-related
+//
+extension ProductFormViewModel {
+    func checkIfBlazeIntroViewIsNeeded() {
+        if let controller = blazeCampaignResultsController, controller.numberOfObjects == 0 {
+            shouldShowBlazeIntroView = true
+        }
+    }
+
+    func updateShouldShowBlazeIntroView(to newValue: Bool) {
+        shouldShowBlazeIntroView = newValue
+    }
```

Comment:
This method is redundant. We only need to toggle the `@Published` property in the view model for SwiftUI view to rerender its view. In UIKit, we dismiss the intro view manually so we don't need to update the value of `shouldShowBlazeIntroView`.

Reply:
Resolved in https://github.com/woocommerce/woocommerce-ios/pull/11011/commits/7cc08fde5ad0a04b3be82451b7473f1468d99409